packages/core/src/controllers/ConfigCtrl.ts

```diff
@@ -13,14 +14,20 @@ const state = proxy<ConfigCtrlState>({
   themeMode: isDarkMode() ? 'dark' : 'light',
   themeColor: 'default',
   themeBackground: CoreUtil.isMobile() ? 'themeColor' : 'gradient',
+  themeZIndex: 89,
```

Comment:
Non Blocking: Can you explain how they overrides the whole zIndex PR you are closing?

Reply:
We default to z-index of `89` but there is now an option to override it. Override happens in `ThemeUtil` file within `setTheme` method (using css var)